src/Controllers/CrudTraits/CrudIndexTrait.php

```diff
@@ -45,8 +45,10 @@ public function index(Request $request)
 
         // SORT COLUMNS ------------------------
         $availableSortColumns = $this->getSortColumns();
-        if (!empty($availableSortColumns) && $request->has('sort')) {
-            foreach ($request->get('sort') as $value) {
+        $sortQuery = $request->get('sort', ['created_at,desc']);
```

Comment:
can we add this as a property or method `defaultSorting` that is empty by default
For current project we want to sort for specific model by created_at desc not all projects and not all models